main.py

```diff
@@ -23,17 +23,17 @@
 ]
 
 # Get Panther-managed rules for onboarded log types
-base_rules = get_panther_rules(LogTypes=onboarded_log_types)
+base_rules = get_panther_rules(log_types=onboarded_log_types)
```

Comment:
We should change this up to be defined parameters like `.override()` has to avoid this sneaking through in the future

Reply:
Yes please... Can you take that one @maxrichie5 ?